### PR TITLE
Testing many images at once

### PR DESCRIPTION
**Pull request by MarcelTorne**
In this PR, we are bringing the possibility to test several images at once and the results are downloaded into a csv file.

### CODE REVIEW

**Comment by MarcelTorne** [comment]
This PR is done over #53, so 53 has to be merged before this one.

File: mobile-browser-based-version/src/components/building_frames/ImageTestingFrame.vue [hunk 1]
```diff
@@ -274,48 +274,71 @@ export default {
       IMAGE_WIDTH: null,
   
       model_name: "",
-      FILES: [],
+      FILES: {},
```
Comment (PaulMansat) [inline mobile-browser-based-version/src/components/building_frames/ImageTestingFrame.vue:277]: Once this branch is merged, I'll implement the new file upload manager with the new UploadingFrame

**Review by PaulMansat — COMMENTED** [review]
Great, thanks, Marcel! Once these changes are merged with the master branch, I'll bring the file upload manager

**Comment by mmilenkoski** [comment]
The CSV download didn't work on my machine. I pushed a fix. @MarcelTorne can you check that it works correctly on your side as well?

**Comment by MarcelTorne** [comment]
> The CSV download didn't work on my machine. I pushed a fix. @MarcelTorne can you check that it works correctly on your side as well?

This is working on my side too! Thank you very much for the fix Martin!

**Review by mmilenkoski — APPROVED** [review]
thank you Marcel, great work!

**Pull request merged**